libs/blocks/aside/aside.css

```diff
@@ -618,6 +836,34 @@
     flex: 0 0 calc(36% - var(--spacing-s));
   }
 
+  .aside.split.icon-stack .foreground.container .text,
+  .aside.split.bio .foreground.container .text {
+    padding: var(--spacing-xxl) 0;
+  }
+
+  .aside.split .foreground.container .text .icon-stack-area li {
+    max-width: calc(50% - 6px);
+    min-width: calc(50% - 6px);
+  }
+
+  .aside.split .foreground.container .text .icon-stack-area {
+    flex-direction: row;
+  }
+
+  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area {
+    height: 83px;
```

Comment:
These values should be consistent and use out icon-size variable. Where is this unit coming from?

Reply:
@ryanmparrish The height for center aside icon is 83px for desktop and mobile screen sizes as per the XD shared for the enhancement ticket.

Reply:
@ryanmparrish XD team has confirmed that the height should be 80px across screen sizes. I have make the corresponding changes in the PR.